src/models/index.js

```diff
@@ -0,0 +1,40 @@
+
+const fs = require('fs');
+const path = require('path');
+const Sequelize = require('sequelize');
+const envConfigs =  require('../config/config');
```

Comment:
1. You need to use Import/Export modules for your imports
2. Line 5 is not resolvable. check if you're referencing an actual file that exists
3. Run `npm run lint` to see the errors and fix all of them in all the files.